src/routes/UserSettings/EditProfileTab/EditProfileTab.js

```diff
@@ -99,6 +170,13 @@ export default class EditProfileTab extends Component {
   updateSettingDirectly = (key, changed) => value =>
     this.updateSetting(key, changed)({ target: { value } })
 
+  updateSocialSetting = ({ key, value }) => {
+    this.setState({
+      changed: true,
+      edits: { ...this.state.edits, [key]: value }
+    })
+  }
```

Comment:
I think the source of this bug was that `changed` wasn't being set to `true` after users updated their twitter, linkedin, or facebook information.

I added this `updateSocialSetting` function to update `changed` to `true` and update the appropriate value in `edits`. 

`updateSocialSetting` is called in the `SocialControl`'s `handleLinkClick`
<img width="682" alt="Screen Shot 2021-03-24 at 8 37 42 PM" src="https://user-images.githubusercontent.com/38114845/112415264-ca540900-8ce0-11eb-88d7-208350ad0410.png">

Reply:
UPDATE: We're using `updateSettingDirectly` instead of `updateSocialSetting`